server/implementation/src/test/java/io/smallrye/graphql/execution/AsyncTest.java

```diff
@@ -0,0 +1,40 @@
+package io.smallrye.graphql.execution;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import javax.json.JsonObject;
+
+import org.jboss.jandex.IndexView;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test a basic async endpoint
+ * 
+ * @author Phillip Kruger (phillip.kruger@redhat.com)
+ */
+public class AsyncTest extends ExecutionTestBase {
+
+    protected IndexView getIndex() {
+        return Indexer.getTestIndex("io/smallrye/graphql/test/async");
```

Comment:
don't you want to rename it to be more specific (CompletationStage?)

Reply:
Yes :)